src/components/navigation-user/navigation-user.tsx

```diff
@@ -0,0 +1,81 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-navigation-user",
+  styleUrl: "navigation-user.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavigationUser {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteNavigationUserElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted.*/
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the text to display, such as a username or full name.*/
+  @Prop() fullName: string;
```

Comment:
doc: This doc says this could be a username but theres already a username prop.

Reply:
good catch, this needs to be updated to just full name.